src/types/Car.ts

```diff
@@ -137,9 +137,23 @@ export interface Car extends CreatedUpdatedProps {
   userIDs?: string;
   carUsers?: UserCar[];
   type?: CarType;
-  converterType?: string;
+  converter?: Converter;
 }
 
+export interface Converter {
+  powerWatts: number;
+  powerAmps: number;
```

Comment:
You should use here ampPerPhase or ampPhase to follow your way of naming. There is no such a thing like Amperage Power in electricity.